Update to phantomjs 2.1.1

jzaefferer opened the pull request:
This depends on https://github.com/Medium/phantomjs/pull/438 getting merged and published.

I've bumped to major version to reflect the underlying change, skipping 1.x entirely (should've used that previously).

Arkni commented on package.json:
```diff
@@ -1,7 +1,7 @@
   "name": "grunt-lib-phantomjs",
   "description": "Grunt and PhantomJS, sitting in a tree",
-  "version": "0.7.1",
+  "version": "2.0.0",
```
Why not `1.0.0` ?

jzaefferer replied:
That's also fine.

vladikoff commented:
```
npm ERR! No compatible version found: phantomjs@^2.1.1
npm ERR! Valid install targets:
npm ERR! 1.9.19, 1.9.18, 1.9.17, 1.9.16, 1.9.15, 1.9.13, 1.9.12, 1.9.11, 1.9.10, 1.9.9, 1.9.8, 1.9.7-15, 1.9.7-14, 1.9.7-13, 1.9.7-12, 1.9.7-11, 1.9.7-10, 1.9.7-9, 1.9.7-8, 1.9.7-7, 1.9.7-6, 1.9.7-5, 1.9.7-4, 1.9.7-3, 1.9.7-1, 1.9.6-0, 1.9.2-6, 1.9.2-5, 1.9.2-4, 1.9.2-3, 1.9.2-2, 1.9.2-1, 1.9.2-0, 1.9.1-9, 1.9.1-8, 1.9.1-7, 1.9.1-6, 1.9.1-5, 1.9.1-4, 1.9.1-3, 1.9.1-2, 1.9.1-0, 1.9.0-6, 1.9.0-5, 1.9.0-4, 1.9.0-3, 1.9.0-2, 1.9.0-1, 1.9.0-0, 1.8.2-3, 1.8.2-2, 1.8.2-1, 1.8.2-0, 1.8.1-3, 1.8.1-2, 1.8.1-1, 1.8.0-1, 0.2.6, 0.2.5, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.2.0, 0.1.1, 0.1.0, 0.0.9, 0.0.8, 0.0.7, 0.0.6, 0.0.5, 0.0.4, 0.0.3, 0.0.2, 0.0.1
npm ERR! 
```

Arkni commented:
@vladikoff 

For this change to work, Medium/phantomjs#438 should be merged and published which is not the case right now.

vladikoff commented:
@Arkni thanks, sounds good!

seanparmelee commented:
https://github.com/Medium/phantomjs/pull/439 was merged and phantomjs 2.1.1 is now published. Can somebody restart the build?

jzaefferer commented:
Updated. Set the package version to 1.0.0 as suggested here, updated the dependency to `^2.1.2` (that's the latest release). Travis build passed, should be good to land. I can merge myself, but would like for someone else to confirm.

jzaefferer commented:
Are you going to tag and publish, too?

vladikoff commented:
Yeap, one sec

On Tue, Jan 26, 2016 at 9:59 AM, Jörn Zaefferer <notifications@github.com>
wrote:

> Are you going to tag and publish, too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gruntjs/grunt-lib-phantomjs/pull/96#issuecomment-175059243>
> .
>
